elyra/airflow/bootstrapper.py

```diff
@@ -328,20 +343,36 @@ def execute(self) -> None:
                 f"executing python script using " f"'python3 {python_script}' to '{python_script_output}'"
             )
             t0 = time.time()
-            with open(python_script_output, "w") as log_file:
-                subprocess.run(["python3", python_script], stdout=log_file, stderr=subprocess.STDOUT, check=True)
+
+            run_args = ["python3", python_script]
+
+            logger.info(f"Processing file: {python_script}")
```

Comment:
@kevin-bates using logger.info everywhere now. What do you thing about f strings in logging, pros and cons?

https://www.reddit.com/r/learnpython/comments/xx14jw/why_not_use_fstrings_in_logging/

Reply:
I remove f style formatting in my new logging code, but leaving it in the old code ...

Reply:
I find f-strings a bit more readable.  I understand the arguments against using them where the "formatting" happens unconditionally.  In those cases, where it might be expensive to format the data, I would either forgo f-strings or use the logger boolean that indicates if that level of logging is enabled and wrap the "expensive" formatting.  In the grand scheme of things, especially given Elyra's background nature, I think it would be rare that such a need arises.  That said, its always good to count your microseconds.